gitlint/configs/config.yaml

```diff
@@ -180,9 +180,12 @@ yaml:
   extensions:
   - .yaml
   - .yml
-  command: yaml_linter.py
+  command: yamllint
+  arguments:
+  - "--config-data"
+  - "relaxed"
   filter: "(?P<message>.+)$"
```

Comment:
For stylistic messages you should be matching using the line number as well, so you don;t see the messages of lines you didn't change. For structural errors (invalid file) you don't care about the line number, so you always get the error.